pkg/pillar/types/zedroutertypes.go

```diff
@@ -3200,6 +3200,8 @@ type AppInstMetaDataType uint8
 const (
 	AppInstMetaDataTypeNone AppInstMetaDataType = iota // enum for app inst metadata type
 	AppInstMetaDataTypeKubeConfig
+	AppInstMetaDataTypeCurrentVersion
```

Comment:
Is `AppInstMetaDataTypeCurrentVersion` being used?

Reply:
it's not. I confused myself into thinking I need both of these fields. Will remove